pax-runtime/src/engine/expanded_node.rs

```diff
@@ -319,7 +328,12 @@ impl ExpandedNode {
         let parent = borrow!(self.parent_expanded_node).upgrade().unwrap();
         let parent_bounds = parent.layout_properties.bounds.clone();
         let parent_transform = parent.layout_properties.transform.clone();
-        let (transform, bounds) = compute_tab(&self, parent_transform, parent_bounds);
+        let (transform, bounds) = compute_tab(
+            &self,
+            parent_transform,
+            parent_bounds,
+            self.size_fallback.clone(),
```

Comment:
why do we need to include size_fallback as an argument when it is on self?

Reply:
Oops! 🧹